### PR TITLE
[behavior] Centralize test xsl

### PR DESCRIPTION
**Pull request by kangxu**
- Remove unused test xsl resources including:
  usecase-embedding-android-tests
  usecase-litewrt-android-tests

- Update suite.json to centralize test xsl

https://crosswalk-project.org/jira/browse/XWALK-4464

### CODE REVIEW

**Comment by zqzhang** [comment]
These resources are really referenced:

```
behavior$ grep 'testcase.xsl' -r *                                                                                           
suite.json:        "testcase.xsl",
tests.full.tct.xml:<?xml-stylesheet type="text/xsl"  href="testcase.xsl"?>
tests.full.xml:<?xml-stylesheet type="text/xsl"  href="testcase.xsl"?>
tests.tizen.xml:<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
```

**Comment by zqzhang** [comment]
```
usecase-embedding-android-tests$ grep 'testcase.xsl' -r *                                       
tests.android.xml:<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
tests.full.xml:<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
```

**Comment by zqzhang** [comment]
```
usecase-litewrt-android-tests$ grep 'testcase.xsl' -r *                                                   
suite.json:        "testcase.xsl",
tests.android.xml:<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
tests.full.xml:<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
```

**Comment by kangxu** [comment]
@zqzhang  I will add configure in https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/tools/resources/resources.json

**Comment by zqzhang** [comment]
Not to the point.

`tools/resources/resources.json` is only referenced by `tools/build/copy.py`; which has nothing to do with test suite packaging.

We need to make sure that, however, these resource files are successfully built into released test suite packages.

File: behavior/suite.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 "res/images": "res/images",
                 "res/js": "res/js",
                 "res/media/khronos/red-green.theora.ogv": "res/media/red-green.theora.ogv",
-                "testresult.xsl": "res/css/testresult.xsl",
+                "PACK-TOOL-ROOT/resources/xsl/testresult.xsl": "res/css/testresult.xsl",
```
Comment (zqzhang) [inline behavior/suite.json]: Here needs the `tests.css` too. Please keep here all the 3 files.

**Comment by zqzhang** [comment]
Seems this PR is for behavior test suite only, please update the commit message.

**Comment by kangxu** [comment]
@zqzhang  I have updated it following your comments

**Pull request merged**